### DISCUSSION TITLE
The viscoplatic model implemented in MOOSE

### DISCUSSION THREAD

**Discussion by lixinjie-1996** [idaholab/moose#25554] [Q&A General]
Does anyone know which model exactly is implemented in the material class FiniteStrainHyperElasticViscoPlastic? Is there any documentation referred to?

**Comment by GiudGiud** [comment]
Hello

so the object duocumentation is not great
https://mooseframework.inl.gov/source/materials/FiniteStrainHyperElasticViscoPlastic.html
but there's doxygen documentation
https://mooseframework.inl.gov/docs/doxygen/modules/classFiniteStrainHyperElasticViscoPlastic.html

and this page has more details too
https://mooseframework.inl.gov/modules/tensor_mechanics/Stresses.html

Guillaume

**Reply by lixinjie-1996** [reply]
Thank you so much for your reply. it helps!

**Reply by lixinjie-1996** [reply]
But is there any bibliography that explains more about the constitutive models? In all three documentation, there is no specific definition of 'flow rate'. After reading the documentation I still don't really understand how to adapt this model to my material. Could you please provide more references?
